.changeset/tender-ads-learn.md

```diff
@@ -0,0 +1,167 @@
+---
+"@salt-ds/core": patch
```

Comment:
Split this into 2 files, talk about 2 packages separately

Reply:
Done!